pkg/apis/application/v1alpha1/types.go

```diff
@@ -1578,7 +1578,7 @@ func validatePolicy(proj string, role string, policy string) error {
 	}
 	// resource
 	resource := strings.Trim(policyComponents[2], " ")
-	if resource != "applications" {
+	if resource != "applications" && resource != "repositories" && resource != "clusters" {
 		return status.Errorf(codes.InvalidArgument, "invalid policy rule '%s': project resource must be: 'applications', not '%s'", policy, resource)
```

Comment:
Please update error message as well:
```suggestion
		return status.Errorf(codes.InvalidArgument, "invalid policy rule '%s': project resource must be: 'applications', 'repositories' or 'clusters', not '%s'", policy, resource)
```